src/phases/egg-hatch-phase.ts

```diff
@@ -1,5 +1,5 @@
 import BattleScene, { AnySound } from "#app/battle-scene.js";
-import { Egg, EGG_SEED } from "#app/data/egg.js";
+import { Egg } from "#app/data/egg.js";
```

Comment:
nit: do the imports in this file have to have .js at the end?

Reply:
```suggestion
import { Egg } from "#app/data/egg";
```
NIT: Echoing what Temp pointed out, should definitely take the chance to remove `.js` from any imports if you're in the area / changing imports